ml-agents/mlagents/trainers/learn.py

```diff
@@ -268,26 +308,36 @@ def from_argparse(args: argparse.Namespace) -> "RunOptions":
           configs loaded from files.
         """
         argparse_args = vars(args)
+        full_config = {}
```

Comment:
I think we should be clear on what config we are talking about.  I think a better name here would be `run_options_dict` or something.